tests/config/pytest_modules.yml

```diff
@@ -441,9 +441,6 @@ subworkflows/fastq_align_chromap:
 subworkflows/fastq_contam_seqtk_kraken:
   - subworkflows/nf-core/fastq_contam_seqtk_kraken/**
   - tests/subworkflows/nf-core/fastq_contam_seqtk_kraken/**
-subworkflows/fastq_create_umi_consensus_fgbio:
-  - subworkflows/nf-core/fastq_create_umi_consensus_fgbio/**
-  - tests/subworkflows/nf-core/fastq_create_umi_consensus_fgbio/**
```

Comment:
excuse my ignorance, but I think this is done since tests are moved to the sub-workflow src directory.  Is that right?